db/schema.rb

```diff
@@ -509,7 +509,9 @@
       master_users.device_created_at,
       master_users.created_at,
       master_users.updated_at,
-      master_users.deleted_at
+      master_users.deleted_at,
+      master_users.role,
+      master_users.organization_id
```

Comment:
This diff has erupted for a few reasons put together.

1. The [view](https://github.com/simpledotorg/simple-server/blob/master/db/views/users_v01.sql) selects all fields from the `master_users` table at the time the view was generated.
1. A couple fields were added to `master_users` afterwards, and these fields were not added to the view at the time
1. The setup instructions and script advise the use of `db:migrate` instead of `db:setup`, so all migrations are rerun, rather than loading the database schema from `schema.rb`.

My question is: Do we want `role` and `organization_id` to be added to the database view? If yes, I will add another migration to refresh the view with the additional fields. In either case, I will switch the setup to use `db:setup` or `db:schema:load`.